wdl/transforms/draft3/src/test/cases/command_syntaxes.wdl

```diff
@@ -9,6 +9,7 @@ task a {
   command {
     echo ~{"hello"} ${world1}
     echo goodbye ~{world2}
+    echo ~{sep=", " true="--yes" false="--no" default="foo" world1}
```

Comment:
Hmm, I'm actually a bit disappointed that the `toWom` test hasn't complained about invalid attributes for this `String` placeholder value....

Reply:
WOMp WOMp